sysl2/sysl/strSet_test.go

```diff
@@ -0,0 +1,207 @@
+package main
+
+import (
+	"testing"
+
+	"github.com/anz-bank/sysl/src/proto"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestMakeStrSet(t *testing.T) {
+	a := MakeStrSet("a", "b", "c", "e")
+	assert.Equal(t, 4, len(a), "Unexpected result")
```

Comment:
`, "Unexpected result"` is redundant. Ditto below.

Reply:
okay. remove all of them.